cgmanifest.json

```diff
@@ -6726,6 +6736,26 @@
         }
       }
     },
+    {
+      "component": {
+        "type": "other",
+        "other": {
+          "name": "influxdb2",
+          "version": "2.4.0",
+          "downloadUrl": "https://github.com/influxdata/influxdb/archive/refs/tags/v2.4.0.tar.gz"
+        }
+      }
+    },
+    {
+      "component": {
+        "type": "other",
+        "other": {
+          "name": "influx-cli",
+          "version": "2.4.0",
+          "downloadUrl": "https://github.com/influxdata/influx-cli/archive/refs/tags/v2.4.0.tar.gz"
+        }
+      }
+    },
```

Comment:
Here also use alphabetical order. And there's also a script to help you out with adding new entries:
```bash
toolkit/scripts/update_cgmanifest.py last cgmanifest.json SPECS/{flux,influx*}/*.spec
```